src/7factor.io/converter/ecs.go

```diff
@@ -14,16 +13,118 @@ type Pair struct {
 	Value string `json:"value"`
 }
 
+
 func TransformAndTranslate(contents string) (string, error) {
 	if contents == "" {
 		return `[]`, errors.New("contents cannot be empty")
 	}
 
-	cleanedContents := cleanContents(contents)
-	pairs := transform(cleanedContents)
+	var itemsToBeParsed []string
```

Comment:
hmmmm me thinks this loop should be its own function. Want keep the purity of what `TransformAndTranslate` does (two distinct actions as the name obviously implies)